README.md

```diff
@@ -164,7 +161,7 @@ saving secret token on your client.
 App.S3UploadComponent = EmberUploader.FileField.extend({
   url: '',
 
-  filesDidChange: (function() {
+  filesDidChange: function(files) {
     var uploadUrl = this.get('url');
     var files = this.get('files');
```

Comment:
@poteto No need to get files again since `files` are already passed as argument.